Makefile.shared

```diff
@@ -9,7 +10,10 @@ ifeq ($(USE_WORKSPACE),YES)
 SCHEME ?= $(notdir $(CURDIR))
 WORKSPACE_PATH ?= $(dir $(lastword $(MAKEFILE_LIST)))WebKit.xcworkspace
 XCODE_TARGET = -workspace $(WORKSPACE_PATH) -scheme "$(SCHEME)"
-XCODE_OPTIONS += WK_VALIDATE_DEPENDENCIES=YES_ERROR
+# FIXME: Move this setting to xcconfigs once all local Xcode builds of WebKit
+# happen in the workspace. When this is only passed on the command line, it
```

Comment:
How far are we from this being the case?

Reply:
Quite close! Once Make uses workspace builds by default, engineers will need to go out of their way to build outside of a workspace.